scripts/exgfs_wave_post_pnt.sh

```diff
@@ -252,7 +252,9 @@ source "${USHgfs}/preamble.sh"
                                ww3_outp_spec.inp.tmpl > ww3_outp.inp
 
     ${NLN} mod_def.$waveuoutpGRD mod_def.ww3
-    HMS="${cyc}0000"
+    #export OFFSET_START_HOUR=$( printf "%02d" ${half_assim} )
+    sec=$( printf "%02d" $(( cyc + OFFSET_START_HOUR )) )
+    HMS="${sec}0000"
```

Comment:
Using `sec` for an hour is extremely confusing
```suggestion
    hh=$( printf "%02d" $(( cyc + OFFSET_START_HOUR )) )
    HMS="${hh}0000"
```